scipy/signal/_splinemodule.cc

```diff
@@ -1,8 +1,12 @@
+#include "Python.h"
+#include "numpy/arrayobject.h"
+#include <complex>
+#include<cmath>
 #include "_splinemodule.h"
 
-#define PyArray_MIN(a,b) (((a)<(b))?(a):(b))
 
-static void
+
+void
 convert_strides(npy_intp* instrides,npy_intp* convstrides,int size,int N)
```

Comment:
Why are we dropping `static` here? (162ed2d602240bb291b0a927ea7f94426789b36b)

Reply:
My impression was that `static` was considered not recommended in C++ as opposed to C, but even if it is, it's not worth splitting hairs on; so it's back.